client/components/search/search.css

```diff
@@ -0,0 +1,49 @@
+.search-form {
+  display: inline-block;
+  position: relative;
+}
+
+.search-form-input {
+  width: 100%;
+  height: 32px;
+  line-height: 28px;
+  margin: 0;
+  padding: 10px 16px 10px 16px;
+  border: 1px solid #DFDFDF;
+	border-top: 1px solid #b3b3b3; 
+	padding-top: 7px;
+	padding-bottom: 8px;
+  border-radius: 7px;
+  background-color: white;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  font-family: Tahoma, sans-serif;
+  font-size: 13px;
+  outline: none;
+}
+
+.search-form-input:focus {
+ border: 1px solid #CCDFFF; 
+}
+
+.search-form-btn {
```

Comment:
image on search button does not match Figma and does not centered 
![image](https://user-images.githubusercontent.com/10188652/212284701-abc616bb-315a-480c-8b1a-666010769804.png)